PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,111 @@
+# Description
+> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context and any links to documentation created.
+> Reference: [PR Submission Guide](/submission_process.md)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+### Dependencies
+> List all libraries and imports utilized, e.g. Ownable.sol from OZ
+
+### Externalities
+> List all external (to this PR) contracts that are interacted with - within Yield ecosystem and without.
+
+> _Note to Reviewer: Examine every component the feature may touch. Then go back through excluded components and ensure they cannot be affected._
+
+### Risk 
+> Highlight areas of risk. E.g. Assembly code blocks, use of unchecked, function call made to an external contract that lies outside of the Yield ecosystem. 
+
+> Example: fn A in contractB.sol makes a call to someexternalcontract.sol
+
+## Contracts
+> This section serves as a brief for the reviewer, code author is to introduce contracts and elaborate on salient points.
+
+> ContractA.sol -> description + points
```

Comment:
idk how i feel about all these examples.  We'll leave them in for now ig, but we might end up removing it.   In addition to them cluttering up the page, they might be too opinionated and not appropriate for all prs